### DISCUSSION TITLE
Why highlight.js instead of Prism?

### DISCUSSION THREAD

**Discussion by Nantris** [mantinedev/mantine#5012] [Q&A] [answered]
I understand 7.x deprecates Prism but I haven't been able to find any discussion of why Highlight.js was chosen over Prism. I'd love to know the reasoning.

**Comment by rtivital** [comment, marked as answer]
Initially, I thought that highlight.js has better tsx syntax highlighting, but unfortunately that is not the case. They both have major issues, so there is not much difference – I've decided to keep highlight.js since there are no better options.
